### ISSUE TITLE
Topic Suggestions for Initial Batch of Articles

### ISSUE THREAD

**Issue by RemLampa** [freeCodeCampManila/blog#5]
Any ideas on what our members can write about in our blog's first release?

**Comment by jimsulit** [comment]
Hmm, here's mine. Since we're newbie friendly naman, maybe it's better to discuss newbie obstacles/roadblocks? Better if we can include concrete examples from our experiences and of course what we did or what we are currently doing to overcome those. This could be conceptual or they could talk about the programs they are working on where they found the obstacles/roadblocks

**Comment by arantebw** [comment]
I have an idea on providing the very basic tutorials on certain tools, terminologies, best-practices that we use and should be at least familiar with as developers. Let say, "An Introduction to Markdown", where in it will describe the very basic of what it is and its purpose, how we use it, and how to create it. I thought this is a great way to provide assistance to the newbies and at the same time mastering any particular tools, terminologies, and best-practices as a developer.

**Comment by RemLampa** [comment]
@arante Yes! That's a great idea. Then you can link to the blog post from the documentation too!

@jimsonsulit Yup. I was also hoping we can work more on those types of topics.

**Comment by RemLampa** [comment]
Also, topics don't have to be in English. They can be written in Tagalog to add local flavor to our blog.

**Comment by arantebw** [comment]
@ibleedfilm Honestly, yun talaga ang first article na gusto kong gawin, "An Introduction to Markdown". Pwede ko ng simulan yung draft?

**Comment by jimsulit** [comment]
@arante magandang intro yan! Go na with that and let me know how we can help. :)

**Comment by RemLampa** [comment]
@arante Yes, by all means. Haha. I forgot to mention yesterday, no one has to ask permission to start work on his/her contribution.

There is an approval process anyway upon submission of your pull request.

I suggest you open a new issue with your chosen topic, which we can close once merged with staging branch. :)

**Comment by nyel-gh** [comment]
I agree with @arante. Para sa mga newbie na katulad ko, malaking tulong yung pag provide ng information tungkol sa mga tools at best practices.

**Comment by jromest** [comment]
Hi, I would like to contribute. It will be beneficial for us beginners to practice Git and start contributing. 

_Buhayin ang repo na to!_ :fist_oncoming:
